ansible_mitogen/transport_config.py

```diff
@@ -475,12 +475,13 @@ def ansible_ssh_timeout(self):
         )
 
     def ssh_args(self):
+        local_vars = self._task_vars.get("hostvars", {}).get(self._inventory_name, {})
         return [
             mitogen.core.to_text(term)
             for s in (
-                C.config.get_config_value("ssh_args", plugin_type="connection", plugin_name="ssh", variables=self._task_vars.get("vars", {})),
-                C.config.get_config_value("ssh_common_args", plugin_type="connection", plugin_name="ssh", variables=self._task_vars.get("vars", {})),
-                C.config.get_config_value("ssh_extra_args", plugin_type="connection", plugin_name="ssh", variables=self._task_vars.get("vars", {}))
+                C.config.get_config_value("ssh_args", plugin_type="connection", plugin_name="ssh", variables=local_vars),
+                C.config.get_config_value("ssh_common_args", plugin_type="connection", plugin_name="ssh", variables=local_vars),
+                C.config.get_config_value("ssh_extra_args", plugin_type="connection", plugin_name="ssh", variables=local_vars)
             )
             for term in ansible.utils.shlex.shlex_split(s or '')
```

Comment:
Note to self: `for term  in ansible.utils.shlex.shlex_split(s or '')` probably behaves identically to `for term  in ansible.utils.shlex.shlex_split(s)`. An experiment/test would confirm this. If true, then I think `MitogenViaSpec.ssh_args()` and `PlayContextSpec.ssh_args()` are functionally identical. I think this PR would preserve this property.